src/win/fs.c

```diff
@@ -1403,13 +1404,18 @@ static void fs__fchmod(uv_fs_t* req) {
 }
 
 
-INLINE static int fs__utime_handle(HANDLE handle, double atime, double mtime) {
-  FILETIME filetime_a, filetime_m;
+INLINE static int fs__utime_handle(HANDLE handle, double btime, double atime, double mtime) {
+  FILETIME filetime_b, filetime_a, filetime_m;
+  int btime_provided = !isnan(btime);
+
+  if (btime_provided) {
+    TIME_T_TO_FILETIME((time_t) btime, &filetime_b);
```

Comment:
Followed convention again.  If we change this for `btime`, we'd need to do the same for `atime` and `mtime` which were already using this pattern.